[FIX] *type*: check company, use Domain object

jjscarafia opened the pull request:
Por si account_ux no esta instalado, usamos Domain ya que el check ompany domain de journals de odoo devuelve lista

jjscarafia committed:
[FIX] *type*: check company, use Domain object

Por si account_ux no esta instalado, usamos Domain ya que el check ompany domain de journals de odoo devuelve lista

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/sale/pull/1486.png)](https://mergebot.adhoc.ar/ingadhoc/sale/pull/1486)

jjscarafia committed:
[FIX] _type_: fix company change and wrong constraint

1. La constraint ya no es necesaria, ya estamos chequando de manera correcta el diario de pago
2. arreglamos mandar la invoice company que lo había borrado por error

jjscarafia commented:
@roboadhoc r+

roboadhoc commented:
@jjscarafia because this PR has multiple commits, I need to know how to merge it:

* `merge` to merge directly, using the PR as merge commit message
* `rebase-merge` to rebase and merge, using the PR as merge commit message
* `rebase-ff` to rebase and fast-forward
